os/net/lwip/src/include/lwip/lwipopts.h

```diff
@@ -663,6 +663,8 @@
 
 #ifdef CONFIG_NET_SO_SNDTIMEO
 #define LWIP_SO_SNDTIMEO	CONFIG_NET_SO_SNDTIMEO
+#else
+#define LWIP_SO_SNDTIMEO	0
```

Comment:
In os/net/lwip/src/include/lwip/opt.h, there is
```h
#ifndef LWIP_SO_SNDTIMEO
#define LWIP_SO_SNDTIMEO                0
#endif
```
By this conditional, this looks unnecessary. Could you check that?

Reply:
corrected